2b57459b19f333e45e969d6554df35f2  twopi.seamless
2b57459b19f333e45e969d6554df35f2  /tmp/twopi-database.seamless
OUTGOING serve_graph jobmaster
INCOMING jobmaster serve_graph
OUTGOING serve_graph jobmaster
ADD SERVANT buffer
ADD SERVANT buffer_info
ADD SERVANT buffer
ADD SERVANT semantic_to_syntactic
ADD SERVANT buffer_info
ADD SERVANT transformation
ADD SERVANT semantic_to_syntactic
INCOMING jobmaster serve_graph
connection handler failed
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/site-packages/websockets/legacy/server.py", line 232, in handler
    await self.ws_handler(self)
  File "/opt/conda/lib/python3.10/site-packages/websockets/legacy/server.py", line 1146, in _ws_handler
    return await cast(
  File "/seamless/seamless/communion_server.py", line 283, in _serve_outgoing
    await self._listen_peer(websocket, peer_config)
  File "/seamless/seamless/communion_server.py", line 226, in _listen_peer
    protocol_message = await websocket.recv()
  File "/opt/conda/lib/python3.10/site-packages/websockets/legacy/protocol.py", line 553, in recv
    await self.ensure_open()
  File "/opt/conda/lib/python3.10/site-packages/websockets/legacy/protocol.py", line 930, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: received 1000 (OK); then sent 1000 (OK)
Set up a communion outgoing port 8602, listening on 0.0.0.0
REMOVE SERVANT buffer
REMOVE SERVANT buffer_info
REMOVE SERVANT semantic_to_syntactic
703158b22572ca9204d53508f97e8aab  twopi-colored-jobmaster.seamless
